packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/Tooltip.tsx

```diff
@@ -109,13 +151,13 @@ export default function Tooltip(props: TooltipProps) {
       // set the content
       switch (data.format) {
         case 'single': {
-          const d = formatSingle(
-            formatter,
-            data.total,
-            sampleRate,
-            numTicks,
-            units
-          );
+          const newLeftContent = {
+            percent: formatPercent(data.total / numTicks),
+            samples:
+              units === 'trace_samples' ? '' : numberWithCommas(data.total),
+            units,
+            formatedValue: formatter.format(data.total, sampleRate),
```

Comment:
Is this calculation valid for tracing? Asking because sampleRate has some bogus value here
 https://github.com/pyroscope-io/pyroscope/blob/main/packages/pyroscope-flamegraph/src/convert/convert.ts#L17